config/locales/devise.two_factor_authentication.en.yml

```diff
@@ -3,7 +3,7 @@ en:
     errors:
       messages:
         select_two_factor: You must select a two factor authenticaton preference.
-        user_not_authenticated: Access denied. Please confirm your one-time passcode to continue.
+        user_not_authenticated: Please enter your one-time password to continue.
```

Comment:
Good point. I'm fine with removing it. Maybe we'll keep it for now and check in with the design team on our various flashes?